x/auth/client/cli/sign.go

```diff
@@ -77,17 +99,33 @@ func makeSignCmd(cdc *amino.Codec, decoder auth.AccountDecoder) func(cmd *cobra.
 	}
 }
 
-func printSignatures(stdTx auth.StdTx) {
+func printSignatures(stdTx auth.StdTx) bool {
 	fmt.Println("Signers:")
-	for i, signer := range stdTx.GetSigners() {
+	signers := stdTx.GetSigners()
+	for i, signer := range signers {
 		fmt.Printf(" %v: %v\n", i, signer.String())
 	}
+
+	sigs := stdTx.GetSignatures()
 	fmt.Println("")
 	fmt.Println("Signatures:")
+	success := true
+	if len(sigs) != len(signers) {
+		success = false
```

Comment:
Should we return or log a specific error here?

Reply:
Well, we could. Not urgent though I think